pkg/config/config.go

```diff
@@ -314,21 +313,16 @@ func (c *Config) encryptContextStateTokens(tempAuthToken, tempAuthRefreshToken s
 		c.Context().GetState().Nonce = nonce
 	}
 
-	if regexp.MustCompile(authTokenRegex).MatchString(tempAuthToken) {
+	// We prefix encrypted tokens with "AES/GCM/NoPadding" to ensure that they are only encrypted once.
+	if !strings.HasPrefix(tempAuthToken, secret.AesGcm) && tempAuthToken != "" {
```

Comment:
Should we add a prefix for Windows, too? I think the auth token regex was really the only guard against accidental double encryption on Windows.

Reply:
Yeah it would be great to stay consistent, but we would need to make sure not to break Windows users' CLIs

Reply:
As an aside, I don't think this was a great pattern. Instead of writing logic to avoid double encryption, we should have just made sure encrypt was only called once.